### DISCUSSION TITLE
automatic memory allocation, allows for rendering at high res and/or high speed (example rendered at 1024x2816 in one pass, info inside)

### DISCUSSION THREAD

**Discussion by dudumaroja** [hlky/stable-diffusion-webui#693] [Feature suggestions]
Found this post in the SD subreddit? can this be implemented?

https://www.reddit.com/r/StableDiffusion/comments/x6dhks/fork_for_automatic_memory_allocation_allows_for/

i could generate a 1024x512 image on a 1660 ti in 3min (first try.. will try larger formats later ) 

a dragon using knght armor seed:2114908983 width:1024 height:512 steps:50 cfg_scale:7.5 sampler:k_lms 

edit again:

Maximazed sliders!!!!!!!!!!!!!!!!!!!
strider cuting a flying robot, ninja, videogame, ray tracing, unreal engine 5, seed:1788584166 width:1024 height:1024 steps:50 cfg_scale:7.5 sampler:k_lms

Took 488.74s total (488.74s per image) Peak memory usage: 6083 MiB / 6144 MiB / 98.993%

**Comment by Muskelmagier** [comment]
Yeah, tried it also myself and just replaced the attention.py works fine
